### ISSUE TITLE
Proton 9.0-4 breaks LaLa and Appimage does not work

### ISSUE THREAD

**Issue by err0r2k** [wyyadd/LaLa#64]
After an update to Proton 9.0-4 it is not possible anymore to start a Trainer.
I use the Binary version of LaLa because AppImage does not work.
LaLa is starting and it is possible to select a game.
With Proton 9.0-4 -> after a click on start, the trainer will not launch.
With Proton 8.0.-5 -> the trainer will launch without problems.
I also cleaned the cache.

AppImage will not start.
`chmod +x LaLa_linux_amd64.AppImage && \
./LaLa_linux_amd64.AppImage`

Starting the Appimage in console:
```
Gtk-Message: 17:24:19.180: Failed to load module "colorreload-gtk-module"
Gtk-Message: 17:24:19.181: Failed to load module "window-decorations-gtk-module"

(LaLa:21272): Gtk-WARNING **: 17:24:19.209: Could not load a pixbuf from /org/gtk/libgtk/icons/16x16/status/window-minimize-symbolic.symbolic.png.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:495:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Format der Bilddatei unbekannt (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:495:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Format der Bilddatei unbekannt (gdk-pixbuf-error-quark, 3)
```

**Comment by cnschu** [comment]
can it be that the project is abandoned?
that would be sad,

**Comment by err0r2k** [comment]
Unfortunately this could be possible. Very sad. :-(
My solution to get the the App and Trainer to work in games is to use GE-Proton9-13.
With this version, I have no problems to start the Trainer in several games.

**Comment by cnschu** [comment]
And how do you do that?
I cant select the version you mentioned in steam.

**Comment by err0r2k** [comment]
> And how do you do that? I cant select the version you mentioned in steam.
> 
> ![Image](https://github.com/user-attachments/assets/c8080e24-c8b8-44d5-8d47-a2bc464a7e24)

You need the Tool ProtonUp-Qt. You can download it as native package or as Flatpak. I don't know which Linux distribution/Desktop you are using.
After downloading you open the tool and choose Steam from the drop-down list. Then click on add and choose GE-Proton9-13.

![Image](https://github.com/user-attachments/assets/11423bf6-491f-4dbc-a0b3-018d10e7b8c0)

What is GE-Proton? [GE-Proton](https://github.com/GloriousEggroll/proton-ge-custom) is a Compatibility tool for Steam Play based on Wine and additional components.
In Short: Proton experimental with additional components, which can fix some problems in games.

**Comment by cnschu** [comment]
thanks, works great.

**Comment by wyyadd** [comment]
> Starting the Appimage in console:
```
Gtk-Message: 17:24:19.180: Failed to load module "colorreload-gtk-module"
Gtk-Message: 17:24:19.181: Failed to load module "window-decorations-gtk-module"

(LaLa:21272): Gtk-WARNING **: 17:24:19.209: Could not load a pixbuf from /org/gtk/libgtk/icons/16x16/status/window-minimize-symbolic.symbolic.png.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:495:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Format der Bilddatei unbekannt (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../gtk/gtk/gtkiconhelper.c:495:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Format der Bilddatei unbekannt (gdk-pixbuf-error-quark, 3)
```

These errors have no impact on the core functionality of lala. You can ignore them.


Duplicate of https://github.com/wyyadd/LaLa/issues/66
See https://github.com/wyyadd/LaLa/issues/66#issuecomment-2848863152 for more details
